content/docs/reference-react-dom-server.md

```diff
@@ -15,31 +15,31 @@ import ReactDOMServer from 'react-dom/server';
 var ReactDOMServer = require('react-dom/server');
 ```
 
-## Overview
+## Resumen
 
-The following methods can be used in both the server and browser environments:
+Los siguientes métodos se pueden utilizar tanto en el servidor como en el entorno del navegador:
 
 - [`renderToString()`](#rendertostring)
 - [`renderToStaticMarkup()`](#rendertostaticmarkup)
 
-These additional methods depend on a package (`stream`) that is **only available on the server**, and won't work in the browser.
+Estos métodos adicionales dependen de un paquete (`stream`) que **solo estan disponibles en el servidor**, y no funcionara en el navegador.
```

Comment:
Taking into account the accents and if you are referring to a single package should be `solo está disponible en el servidor y no funcionará en el navegador`.